### DISCUSSION TITLE
BLTouch auto bed levelling does not probe the whole bed

### DISCUSSION THREAD

**Discussion by Amos343** [Jyers/Marlin#107] [Q&A] [answered]
So whenever an auto bed level starts the 3x3 probe points don't cover the full bed. The left side seems okay, the centre points are shifted left a fair bit and the right points are way over to the left leaving a good 25mm from the right most side of the bed to the probe. 

Having owned a Prusa i3 which probes the whole bed and does not leave a huge gap to the right side, it leaves me wondering if this is normal for the ender 3 v2? Won't this effect the print quality in any way? 

Ive been trying to get my head around editing the template firmware but I have very little experience and really don't know what I am doing

Can anyone please shed some light on this

**Comment by Jyers** [comment, marked as answer]
Please search for your question in discussions before posting a new question: #60
